src/IceRpc.Slice/IProxy.cs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) ZeroC, Inc.
 
+using System.ComponentModel.DataAnnotations;
```

Comment:
Why is this using required?

Reply:
Not sure how this using was added. Will remove.